### PR TITLE
docs: Add more details to the test framework README

### PR DESCRIPTION
**Pull request by p-offtermatt**
## Description

Closes: N/A, minor change

Adds more information about how to run and generate traces for the e2e framework,
as suggested by @sainoe 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary 
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

### CODE REVIEW

File: tests/e2e/README.md [hunk 1]
```diff
@@ -10,4 +10,15 @@ If the chain state from `state.go` is modified, the `ChainStateWithProposalTypes
...
 * add a case for your proposal type to `json_utils.go/UnmarshalProposalWithType`
-* add a generator for your proposal type in `state_rapid_test.go` and add it to the `GetProposalGen` function 
\ No newline at end of file
+* add a generator for your proposal type in `state_rapid_test.go` and add it to the `GetProposalGen` function
+
+## Regenerating Traces
+
+The traces in `tracehandler_testdata` are generated by the test `trace_handlers_test.go/TestWriteExamples`.
+
+You can regenerate them by running `./regenerate_traces.sh/`.
```
Comment (sainoe) [inline tests/e2e/README.md:19]: I think the `regenerate_traces.sh` file wasn't committed

Comment (p-offtermatt) [reply tests/e2e/README.md:19]: Thanks!

**Review by sainoe — APPROVED** [review]
Thanks!

File: tests/e2e/regenerate_traces.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+go test -timeout 30s -run ^TestWriteExamples$ github.com/cosmos/interchain-security/v3/tests/e2e
```
Comment (shaspitz) [inline tests/e2e/regenerate_traces.sh:3]: Seems like this'd be more appropriate as a make target

**Review by shaspitz — APPROVED** [review]
One comment but looks solid!

File: tests/e2e/regenerate_traces.sh (hunk 2 above)
Comment (p-offtermatt) [reply tests/e2e/regenerate_traces.sh:3]: True, good call

**Pull request merged**